include/core/CBoostJsonWriterBase.h

```diff
@@ -134,6 +131,14 @@ class CBoostJsonWriterBase {
         return this->getAllocator()->get();
     }
 
+    void removeAllocator(const std::string& allocatorName) {
+        if (m_AllocatorCache.find(allocatorName) != m_AllocatorCache.end()) {
+            TPoolAllocatorPtr allocator = m_AllocatorCache[allocatorName];
```

Comment:
It looks slightly inefficient to do `cache.find(key)` followed by `cache[key]` and `erase(key)`. It does the map lookup 3x.

Why not do:
```
allocator = cache.find(name);
if (allocator != cache.end()) {
  allocator->reset();
  cache.erase(allocator);
}
```

Reply:
Good point. I adjusted the code.